python/tests/conftest.py

```diff
@@ -1484,21 +1484,21 @@ def fixture_aws_credentials() -> Generator[None, None, None]:
 
 @pytest.fixture(name="_s3")
 def fixture_s3(_aws_credentials: None) -> Generator[boto3.client, None, None]:
-    """Mocked S3 client."""
+    """Yield a mocked S3 client."""
     with mock_s3():
         yield boto3.client("s3", region_name="us-east-1")
 
 
 @pytest.fixture(name="_glue")
 def fixture_glue(_aws_credentials: None) -> Generator[boto3.client, None, None]:
-    """Mocked glue client."""
+    """Yiled a mocked glue client."""
```

Comment:
```suggestion
    """Yield a mocked glue client."""
```

Reply:
Good catch! Thanks for your review.

Reply:
Any time, I should thank you for picking this up!